docs-search/client-halogen/spago.yaml

```diff
@@ -33,3 +33,12 @@ package:
     - web-html
     - web-storage
     - web-uievents
+  bundle:
+    type: "app"
+    minify: true
+    sourceMaps: true
+    module: Docs.Search.App
+    outfile: "../../output/Docs.Search.IndexBuilder/docs-search-app.js"
+    platform: browser
+    extra_args:
+      - "--alias:punycode=punycode/"
```

Comment:
punycode is a builtin node module. Since we're bundling for browser, `esbuild` complains about it.
The node module is also considered deprecated and recommends using the upstream npm package `punycode`. So its an easy swap-in. 

The extra `/` at the end is how you tell node and esbuild to override a builtin node package with a user-space package.

https://github.com/JordanMartinez/purescript-language-purescript/commit/db4377dea03ba9c5273a93a8368a53f1d87882c1

Reply:
Sounds great! Could you add the above comment as a comment in the source just above this line?